plaidml/BUILD

```diff
@@ -387,7 +387,7 @@ plaidml_cc_binary(
     name = "libplaidml_pub.so",
     linkopts = [
         # Prefer internal library definitions.
-        "-Wl,-Bsymbolic-functions",
+        "-Wl,-Bsymbolic-functions,-T",
```

Comment:
I think the `-T` belongs on the linker script line below.  But I'm a little unclear on why this is necessary -- are you compiling with `--config=linux_x86_64`?  If not, that would explain why it's using your system toolchain.

Reply:
Thanks!